mypy.ini

```diff
@@ -0,0 +1,29 @@
+[mypy]
+python_version = 2.7
+
+ignore_missing_imports = True
+
+show_error_codes = True
+
+follow_imports = silent
+
+mypy_path = stubs/
+
+namespace_packages=True
+explicit_package_bases=True
+
+# strict options
+warn_unused_configs = True
+disallow_any_generics = True
+disallow_subclassing_any = True
+disallow_untyped_calls = True
+disallow_untyped_defs = True
+disallow_incomplete_defs = True
+check_untyped_defs = True
+disallow_untyped_decorators = True
+no_implicit_optional = True
+warn_redundant_casts = True
+warn_unused_ignores = True
+warn_return_any = True
+no_implicit_reexport = True
+strict_equality = True
```

Comment:
Everything is enabled(or strict) as of now. So todo is not required.

Reply:
Ah, I see, isn't there one flag to enable all strict checks?

Reply:
Yes, there is a flag. But I chose to move forward with individual configurations because:
1. It will be easy to set a couple of flags off in the future.
2. The configuration file itself gives the information of the which thing is allowed or not.

Reply:
1. Why would we want to disable something in the future?
2. Makes sense.

But one problem I see is that if new strict checks are added to the MyPy library and we would upgrade, we can easily forget about these new strict checks.

Reply:
1. In future while adding type annotations, a case may arise in which we have to ignore some errors. 
But yes, I get your point that this can be changed later.
If mypy introduces new checks, we would not automatically want that check to be added. 
Therefore I think it is better to write all the configurations explicitly. What do you think? Thanks!

Reply:
I think we actually want all new strict check to be enabled. Since we will only upgrade mypy manually we can then see that some mypy checks are failing and disable the strict checks temporarily until we fix it in our codebase and then enable all checks again.

Reply:
So I am replacing all the configurations by strict=True.

Reply:
Done.